Fix misalignment of image and description text

j6k4m8 opened the pull request:
...in the about-me section.

Bootstrap fills a `row` with `header`s at small screen sizes, hence the issue we saw with the old CSS hack.

Before:
![image](https://cloud.githubusercontent.com/assets/693511/7799925/24bdfa40-02db-11e5-8ea7-03398728c840.png)

After:
![image](https://cloud.githubusercontent.com/assets/693511/7799949/7ead5618-02db-11e5-8e61-99764343dd06.png)

j6k4m8 committed:
Fix misalignment of image and description text

...in the about-me section.

Bootstrap fills a `row` with `header`s at small screen sizes, hence the issue we saw with the old CSS hack.

j6k4m8 committed:
Add correct resizing behavior

j6k4m8 committed:
Update main.css

j6k4m8 commented:
Still kinda hard to read. For that fix, see #19.

jovo commented:
did you see that i changed the fonts to use the jhu ones?
also i updated the text.
what do you think?

On Sun, Jan 31, 2016 at 11:17 AM, Jordan Matelsky notifications@github.com
wrote:

> Closed #20 https://github.com/jovo/jovo.github.com/pull/20.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jovo/jovo.github.com/pull/20#event-533138760.

## 

the glass is all full:  half water, half air.
neurodata.io

j6k4m8 commented:
I'm into it :)

On Sun, Jan 31, 2016 at 11:39 AM joshua vogelstein notifications@github.com
wrote:

> did you see that i changed the fonts to use the jhu ones?
> also i updated the text.
> what do you think?
> 
> On Sun, Jan 31, 2016 at 11:17 AM, Jordan Matelsky <
> notifications@github.com>
> wrote:
> 
> > Closed #20 https://github.com/jovo/jovo.github.com/pull/20.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/jovo/jovo.github.com/pull/20#event-533138760.
> 
> ## 
> 
> the glass is all full: half water, half air.
> neurodata.io
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jovo/jovo.github.com/pull/20#issuecomment-177545847.
